### ISSUE TITLE
DELETE /2/tweets/ rate limit improvement/multi delete option

### ISSUE THREAD

**Issue by adacable** [twitterdev/open-evolution#24]
Hi, sorry, I missed the "please create an issue" line. 

I've made [a PR](https://github.com/twitterdev/open-evolution/pull/23) suggesting implementing a multi delete endpoing similar to GET /2/tweets/ but for DELETE, with the other option being easing the rate limit on DELETE /2/tweets/:id so that people can quickly delete large swathes of their timeline as they currently can(50requests /15m is rather low given it was previously an unlimited call). 

Happy to discuss it here if folk would prefer.

**Comment by Vroomfrondal** [comment]
Hopefully they increase this limit. 

50 tweets per 15 minutes is so low.

**Comment by Vroomfrondal** [comment]
@adacable I recently ran into this exact issue when building out a script to mass delete tweets. 

While I agree, this limit is so low you can't really use it - **I was able to get around this** by requesting "elevated access" on my app inside the twitter developer dashboard. It was free and I was granted access instantly. 

Once you get elevated access, you can use [the twitter-v1 delete-tweet endpoint](https://developer.twitter.com/en/docs/twitter-api/v1/tweets/post-and-engage/api-reference/post-statuses-destroy-id)

A couple of side notes: don't forget to:
 1) install twitter v1 (for me this was npm install twitter) 
 2) Here is an example of calling that endpoint with a tweet ID: 
           ``v1Client.post(`statuses/destroy/${tweetID}.json`, () => {}``
3) I didn't have to refactor everything to v1, just the function using the delete endpoint

**Comment by adacable** [comment]
Yep, it's not solving the actual problem though- this is an issue with the v2 api, requesting an adjustment to it- "use the V1 api" is not a solution. OTOH, I suspect the people monitoring this repository have been fired, and no-one knows it exists so.
